pyscript.core/src/plugins/pyterminal.js

```diff
@@ -0,0 +1,110 @@
+// PyScript py-terminal plugin
+import { hooks } from "../core.js";
+
+// XXX TODO:
+// 1. these imports should be lazy?
```

Comment:
```js
const { Terminal } = await import(/* webpackIgnore: true */ "https://cdn.jsdelivr.net/npm/xterm@5.3.0/+esm");
const { Readline } = await import(/* webpackIgnore: true */ "https://cdn.jsdelivr.net/npm/xterm-readline@1.1.1/+esm");
```

Reply:
top level await *should* work out of the box, this file is lazily imported anyway ... we could also decide to install these as peer dependencies or something and let the bundler bring in everything we need?

Reply:
> we could also decide to install these as peer dependencies or something and let the bundler bring in everything we need?

I don't have any strong opinion about that is the best way to go, happy to do whatever you think it's best